### ISSUE TITLE
Crash happen when the netperf file path does not exist

### ISSUE THREAD

**Issue by jiangzeng01** [dotnet/coreclr#25532]
Change:
Line 34 of file corecl\tests\src\tracing\common\NetPerfFile.cs
Hard code the file path to any path which does not exist, example "F:\NotExistFolder\"

Run Test:
coreclr\tests\src\tracing\eventsource\eventsourcetrace\EventSourceTrace.csproj

Actual:
Crash happened.
Expected:
Exit with error message instead of crash.

**Comment by sywhang** [comment]
I don’t think this is a runtime issue. That test crashes because the test doesn’t catch an IO exception during file creation failure. (Since we where we create that file during text execution is not - and shouldn’t be - some random location) While it’d be nice to fix the test to spit out some more useful error message than just die, I don’t think it’s necessarily important.
